data/map altera.txt

```diff
@@ -4292,56 +4292,56 @@ system Zephyrius
 planet Asuras
 	landscape land/city6
 	description `The Alteran world of Asuras is the closest you have seen to an ecumenopolis, most of the planet is covered in buildings of some sort, though at this point, a significant portion of the "city" is in disrepair. Asuras was one of, if not the most important industrial worlds of ancient Altera, it is the birthplace of the Asuran Replicators, a form of self-replicating robots, aiming to take human form.`
-	description `	The Drak targeted this world, destroying most of its industrial capabilities and deactivating the replicators with a specific frequency wave signal, leaving the planet almost uninhabited and in heavily battle-scarred. Even now, the population has not gotten anywhere near its pre-war state, though the organic population has multiplied. As a result of the depopulation, nature has started to take back the cityscape, resulting in noticeable green patches.`
```

Comment:
First sentence, the "in" is grammatically incorrect. I removed it, which works, alternatively you could write ...and in a heavily battle-scarred state. 
The descriptions are like in a dictionary or travel guide, so you should avoid overly long sentences with context switches. Where possible, I ended overlong sentences, so I will not point this out every time.
With the next sentence, at first I did not understand what you want to say, because you did not refer to the property which is not at pre-war state, then I realized you probably mean the size of the population, so I added this. Population is generally referring to inhabitants or beings, so I did not understand what you were trying to say in the part with "multiplied". Do you mean that the plant-life has flourished?
So, I tried to understand the overall picture, and tried to express that. 
Now, if you meant something different, it would be good if you can rewrite it to make it clearer.

Reply:
Ah, I'm beginning to understand. The term "Replicators" had me a bit confused. I associate "replicators" with the machine that replicates (e.g. the food replicators on the Enterprise), whereas replicated beings I know as Replicants (like in Bladerunner). I assume you chose the term "Asuran Replicators" deliberately, to avoid association with the Bladerunner movie, so I'll go along with it. 
How about this (starting at the end of the first paragraph): 
...of ancient Altera. It is also the birthplace of the Asuran Replicators, a form of sentient, self-replicating robots in humanoid form, which were regarded as equal citizens by the Alturans.
The Drak targeted this world, destroying most of its industrial capabilities and dismantling the entire Asuran Replicator population, their prime target, by a specific radio signal. This left the planet in a battle-scarred and almost depopulated state. And while the organic population has multiplied over time, even today the number of inhabitants is nowhere near its pre-war state. As a result nature has started to reclaim the abandoned areas, indicated by green patches visible throughout the cityscape.
Comment: it would read a bit rounder if the "by a specific radio signal" half-sentence could be dropped, but I believe that is information you want to provide, so I left it in there.

Reply:
The specific radio signal is not a necessity, it may even be good to leave it out, adding some mystery and preventing an information overload. Apart from that, your changes sound really good, though I'll have to see whether it actually fits in the description window. I'd appreciate it if you could commit your latest changes, from the comment above, so I can merge the pull request.

Reply:
Done.